src/style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
don't reset your margins and paddings with `*` selector, it's a bad practice. Better use tag names for that
```suggestion
```

Reply:
done